docs/README.md

```diff
@@ -4,9 +4,16 @@
+| There's a problem with the app - help!                   | File a report to help us improve. Choose "Bug Report" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| I have an idea for the app!                              | Choose "Feature Request" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| How do I build the app myself?                           | See the [development guide](ONBOARDING.md#development).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Who can I talk to about product, engineering, or design? | Reach out to our [functional leads](ONBOARDING.md#point-people).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
I'm fine with this, but we are signing ourselves up for a lot of emails to filter through in this case... should we work with PgMs to think through a better place to direct people? Google Group maybe?

Reply:
I think we should do that, but at this moment we don't have that process.  Once we have that process, we can change this.